internal/txlib/add_remote.go

```diff
@@ -0,0 +1,129 @@
+package txlib
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/transifex/cli/internal/txlib/config"
+	"github.com/transifex/cli/pkg/jsonapi"
+	"github.com/transifex/cli/pkg/txapi"
+)
+
+func AddRemoteCommand(
+	cfg *config.Config,
+	api *jsonapi.Connection,
+	projectUrl,
+	fileFilter string,
+	minimumPerc int,
+) error {
+	parts := strings.Split(projectUrl, "/")
+	for parts[len(parts)-1] == "dashboard" || parts[len(parts)-1] == "" {
+		parts = parts[:len(parts)-1]
+	}
+	if len(parts) < 2 {
+		return fmt.Errorf("invalid project URL '%s'", projectUrl)
+	}
```

Comment:
Why are we using `dashboard`? Is there a specific idea behind this?
Also the issue here is that if I add something different than dashboard, it breaks.